packages/core/src/components/dialog/_dialog.scss

```diff
@@ -107,7 +107,7 @@ $dialog-padding: $pt-grid-size * 2 !default;
     color: $pt-icon-color;
   }
 
-  h5 {
+  h4 {
```

Comment:
it's not good practice to use a tag selector -- this becomes a CSS breaking change.

I think we should refactor this CSS to use a `.pt-dialog-header-title` class in _both_ in 1.x and 2.x to avoid this subtle break.

Reply:
I opened https://github.com/palantir/blueprint/pull/2141